babel.config.js

```diff
@@ -3,7 +3,8 @@ module.exports = api => {
     const isTest = api.env('test');
     const presets = [];
     const plugins = [
-        '@babel/plugin-proposal-optional-chaining'
+        '@babel/plugin-proposal-optional-chaining', 
+        "@babel/plugin-proposal-class-properties"
```

Comment:
Should these be single quotes?

Reply:
oh snap, yeah they should be - will fix now!